bvm/ballerina-core/src/main/java/org/ballerinalang/model/util/serializer/providers/bvalue/BTypeBValueProviders.java

```diff
@@ -200,6 +202,8 @@ public BObjectType toObject(BPacket packet, BValueDeserializer bValueDeserialize
         private static final String PACKAGE_PATH = "packagePath";
         private static final String TYPE_NAME = "typeName";
         private static final String FLAGS = "flags";
+        private static final String REST_FIELD_TYPE = "restFieldType";
+        private static final String SEALED = "sealed";
```

Comment:
Shall we changed this to `CLOSED`? `sealed` is not the term used now. We have to change the places where we have referred to closed records as sealed records.